doc/templates/build-wg-meeting-0000-00-00.md

```diff
@@ -0,0 +1,33 @@
+# Node.js Foundation Build WG Meeting YYYY-MM-DD
+
+[GitHub issue]()
+[Meeting video]()
+[Previous meeting]()
+
+Next meeting: DD MMM YYYY
+
+## Present
+- Michael Dawson (@mhdawson)
+- Johan Bergström (@jbergstroem)
+- João Reis (@joaocgreis)
+- Gibson Fahnestock (@gibfahn)
+- Michele Capra (@piccoloaiutante)
+- Kyle Farnung (@kfarnung)
```

Comment:
That's reasonable, especially if people fill in their own names in the attendance. To be honest the key for me is avoiding messing up spelling people's names!

I think I'll remove the names and add a comment saying "add your names here", I'm hoping people can spell their own names 😁 .